doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -112,6 +112,7 @@ Bluetooth Mesh
   * The Kconfig option :kconfig:option:`CONFIG_BT_MESH_DFU_CLI` to no longer enable the Kconfig option :kconfig:option:`CONFIG_BT_MESH_DFU_METADATA_ON_BUILD` by default.
     The Kconfig option :kconfig:option:`CONFIG_BT_MESH_DFU_METADATA_ON_BUILD` can still be manually enabled.
   * The JSON file, added to :file:`dfu_application.zip` during the automatic DFU metadata generation, to now contain a field for the ``core_type`` used when encoding the metadata.
+  * The Light Lightness Control Server to instantly change the current or target lightness level if the state machine is in or transitioning to one of the states "RUN", "PROLONG" or "STANDBY" when receiving a "Light LC Property Set" message modifying the relevant state's level.
```

Comment:
Should be moved to line 875 I think, under `Bluetooth libraries and services`, and also use a link to the server model like this:

```suggestion
  * The :ref:`bt_mesh_light_ctrl_srv_readme` to instantly change the current or target lightness level if the state machine is in or transitioning to one of the states "RUN", "PROLONG" or "STANDBY" when receiving a "Light LC Property Set" message modifying the relevant state's level.
```